caffe2/operators/elementwise_sum_op.cc

```diff
@@ -124,4 +143,11 @@ A after Sum: [[10.  7. 11.]
         "*(type: Tensor`<float>`)* Second tensor to be added element-wise.")
     .Output(0, "C", "*(type: Tensor`<float>`)* Sum of A and B.")
     .InheritOnnxSchema();
-}
+
+OPERATOR_SCHEMA(SumGradient)
```

Comment:
Could you also update the doc of the Sum operator to show that it supports broadcasting?